tier4/{{cookiecutter.project_slug}}/MAINTAINERS.md

```diff
@@ -21,6 +21,8 @@ This is a list of maintainers for this project. See [CODEOWNERS.md](./CODEOWNERS
 
 ## Contributors
 
+_In order to automatically update the MAINTAINERS.md, you must enter a secret into your Secrets and Variables under Actions within your repository settings. The name of the secret must be PUSH_TO_PROTECTED_BRANCH and the value must be a Personal Access Token with specific permissions. Please follow [this link](https://github.com/CasperWA/push-protected?tab=readme-ov-file#notes-on-token-and-user-permissions) for more information._
```

Comment:
We have TODO comments sprinkled in this file with explainer text on how to fill out each section. I would make this a TODO comment